app-frontend/src/app/pages/library/projects/detail/projectScenes/projectScenes.html

```diff
@@ -14,57 +14,54 @@ <h1 class="h3 page-title">
                 ng-show="!$ctrl.$parent.selectedScenes.size && !$ctrl.isEditingProjectName && !$ctrl.isSavingProjectNameEdit">
           <i class="icon-pencil"></i>
         </button>
-        <div class="color-danger" ng-show="$ctrl.showError">
+        <div class="color-danger" ng-if="$ctrl.showError">
           There was an error renaming the project.
           <button class="btn btn-tiny" type="submit"
                   ng-click="$ctrl.dismissErrorMessage()">
             <i class="icon-cross color-danger"></i>
           </button>
         </div>
-        <div ng-show="$ctrl.isEditingProjectName">
+        <div ng-if="$ctrl.isEditingProjectName">
```

Comment:
Yeah, I also noticed that on the Imports page, `ng-show` sometimes caused weird content flashes for me that switching to `ng-if` seemed to solve, presumably because with `ng-if` the components never get instantiated in the first place. I'm all for preferring `ng-if` wherever possible. The main situation I can think of where `ng-if` wouldn't be preferable is if we need to rapidly create/destroy a complex component, in which case we might prefer `ng-show` instead.